Descriptions fix (#426)
primer/design: content/components/breadcrumbs.mdx

---

> title: Breadcrumb
> description: Breadcrumb displays the current page or context within the site, allowing them to navigate different levels of the hierarchy.

Reviewer: The name of the code component is Breadcrumbs (plural). 

```suggestion
title: Breadcrumbs
description: Breadcrumbs display the current page or context within the site, allowing them to navigate different levels of the hierarchy.
```

## Anatomy

> Breadcrumbs are made of links (indicating parents), dividers, and the current page. The default breadcrumb component can be modified to change the number of items within the chain.

Reviewer: ```suggestion
Breadcrumbs are made of links (indicating parents), dividers, and the current page. The default breadcrumbs component can be modified to change the number of items within the chain.
```

### Breadcrumb length

> By default the breadcrumb component can show up to 10 items within the chain.

Reviewer: ```suggestion
By default the breadcrumbs component can show up to 10 items within the chain.
```
